site/content/en/docs/manual/advanced/analytics-and-monitoring/auto-qa.md

```diff
@@ -1,167 +1,351 @@
 ---
-title: 'Automated QA, Review & Honeypot'
+title: 'Automated QA, Review & Honeypots'
 linkTitle: 'Automated QA'
 weight: 1
 description: 'Guidelines for assessing annotation quality in CVAT automatically'
 ---
 
-In CVAT, it's possible to evaluate the quality of annotation through
-the creation of a **Ground truth** job, referred to as a Honeypot.
-To estimate the task quality, CVAT compares all other jobs in the task against the
-established **Ground truth** job, and calculates annotation quality
-based on this comparison.
+In CVAT, it's possible to evaluate the quality of annotation through the creation
+of a validation subset of images. To estimate the task quality, CVAT compares
+all other jobs in the task against the established **Ground truth** job,
+and calculates annotation quality based on this comparison.
 
 > **Note** that quality estimation only supports
 > 2d tasks. It supports all the annotation types except 2d cuboids.
```

Comment:
> It supports all the annotation types except 2d cuboids.

Probably this note can be deleted here since you've mentioned that inside `Comparisons`

Reply:
I want this information to be easily reachable for users so that they could immediately understand if it's available for their use case. I think, this variant provides better UX than if they needed to read all of this only to find out that something they need is not supported.

Reply:
okay